src/BulkWriteResult.php

```diff
@@ -35,8 +35,7 @@ class BulkWriteResult
     private $isAcknowledged;
 
     /**
-     * @param WriteResult $writeResult
-     * @param mixed[]     $insertedIds
+     * @param mixed[] $insertedIds
```

Comment:
I'm not sure if tooling left this behind, but it seems odd to only have _some_ parameters documented here. Is that intentional?

As-is, I don't think `mixed[]` adds any clarification to a generic `array` type hint, so maybe we just want to remove this entirely. Up to you.

Reply:
I replaced `mixed[]` with a generic `array`, you're right it doesn't make any sense.